crate/cli/src/tests/kms/shared/revoke.rs

```diff
@@ -338,3 +338,23 @@ async fn test_non_revocable_symmetric_key() -> KmsCliResult<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_revoke_secret_data() -> KmsCliResult<()> {
+    let ctx = start_default_test_kms_server().await;
+
+    let secret_id = crate::actions::kms::secret_data::create_secret::CreateKeyAction::default()
```

Comment:
crate::actions::kms::secret_data::create_secret::CreateKeyAction::default() -> CreateSecretDataAction::default() (shorten the import)